src/contexts/build.context.tsx

```diff
@@ -179,6 +189,10 @@ async function selectBuild(dispatch: Dispatch, id: string) {
   });
 }
 
+async function clearBuild(dispatch: Dispatch) {
```

Comment:
could we reuse `selectBuild` accepting `null` as param?

Reply:
Done.